pontoon/contributors/static/js/contributor.js

```diff
@@ -27,19 +32,23 @@ $(function() {
     });
   }
 
-  // Save user name by mouse or keyboard
-  $('.submit').click(function() {
-    if ($(this).css('opacity') === "0") {
-      return;
-    }
+  // Save user profile by mouse or keyboard
+  $('.submit').click(function(e) {
     save();
   });
-  input.keydown(function(e) {
+  $('#profile-form').on('keydown', function(e){
     if (e.which === 13) {
       e.preventDefault();
       save();
     }
   });
+  $('#profile-email').focus(function(e) {
+      e.preventDefault();
+      console.log("Email changed");
+      if($('#profile-email').next('.validation').length == 0) {
+        $('#profile-email').after('<span class=\'validation\' style=\'color:red;margin-left: 10px;\'>Changing email will cause a logout.</span>')
+      }
+  });
```

Comment:
This should be done with CSS instead of JS. I'd put it in a span next to the input and show it on focus using CSS rules.